src/VisualStudio/Core/Impl/RoslynVisualStudioWorkspace.cs

```diff
@@ -49,6 +53,17 @@ internal class RoslynVisualStudioWorkspace : VisualStudioWorkspaceImpl
                     Services.GetRequiredService<IOptionService>().RegisterDocumentOptionsProvider(optionsProvider);
                 }
             }
+
+            // this will register todo list and error list to VS
+            VisualStudioDiagnosticListTable.RegisterAsync(
+                asyncServiceProvider, this,
+                exportProvider.GetExportedValue<IDiagnosticService>(),
+                exportProvider.GetExportedValue<ITableManagerProvider>()).Forget();
+
+            VisualStudioTodoListTable.Register(
```

Comment:
Is this for the TODO list? Can we defer that until the crawler has actually found a TODO?